libs/dal/src/repositories/base-repository.ts

```diff
@@ -163,8 +163,8 @@ export class BaseRepository<T_DBModel, T_MappedEntity, T_Enforcement = object> {
     return await Promise.all(promises);
   }
 
-  async bulkWrite(bulkOperations: any) {
-    await this.MongooseModel.bulkWrite(bulkOperations);
+  async bulkWrite(bulkOperations: any, ordered = false): Promise<any> {
```

Comment:
Mongoose doesn't export the Bulk types. Not sure how we can still type this

Reply:
Let's check if it is a version thing and we need to upgrade the Mongoose version. Found some comments mentioning that with the same problem.

Reply:
I see that they don't fix that also in the latest :( super weird...